spec/features/families/admin_views_all_families_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
+
+feature "Admin can see all the families when in families#index" do
+  context "when click on the 'All' button" do
+    scenario "successfully" do
+      create(:family, active: false, last_name: "Scara")
+      create(:family, active: true, last_name: "Liso")
+
+      visit families_path
+      find("#all_families_button").click
+
+      expect(page).to have_text(I18n.t("families.index.all_families"))
+      expect(page).to have_text("false")
+      expect(page).to have_text("Scara")
+      expect(page).to have_text("true")
```

Comment:
I wouldn't care so much about the assertions on lines 13 and 15, and just leave the assertions on lines 14 and 16

Reply:
Modified in sha `b6c4d17`